tutorials/gastby-blog/src/pages/index.js

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import Header from "../components/Header";
+import { graphql } from "gatsby";
+import { Seo } from "../components/Seo";
+import { BlogPostSection } from "../styles/home.styled";
+import Post, { FeaturedPost } from "../components/Post";
+
+const IndexPage = ({ data }) => {
+  let posts = data.webiny.listPosts.data;
+  const latestPost = posts[0];
+
+  return (
+    <React.Fragment>
+      <Seo title="Holla Seo" />
+      <main style={{ border: "1px solid #fff", height: "100vh" }}>
```

Comment:
Dude you already have 2 methods of creating styles ... let's stick to those instead of introducing other methods. Please use a Styled Component here.

Reply:
Great! I'll fix this too